src/components/WwwFrame/TheBasketBar.vue

```diff
@@ -1,8 +1,8 @@
 <template>
 	<!-- eslint-disable max-len -->
 	<div
-		v-if="count > 0"
-		class="tw-flex md:tw-hidden tw-fixed tw-z-sticky"
+		v-if="count > 0 && !hideBasketBar"
+		class="basket-bar tw-flex md:tw-hidden tw-fixed tw-z-sticky"
```

Comment:
Prob could have fixed things by just adding this class but worked up the no css approach instead. Then realized this class is used for locating this component from the monthly good selector that can show on homepage so had to add it anyway...